web.md

```diff
@@ -0,0 +1,37 @@
+# Web
+
+**Prerequisite knowledge**: 
+
+## Intro
+Web - short for World Wide Web.
+
+No prerequisite knowledge required. If you see this document, you are probably aware of what computers are and that they can communicate with each other via global network of computer networks, called the Internet. If so, you're all set to begin your journey with the web.
```

Comment:
Maybe we could link to HTML or Hypertext here?
Info about prerequisite knowledge can be moved to section above.
This description seems very indirect to, it doesn't tell what Web is.